app/executor.go

```diff
@@ -18,11 +17,11 @@ import (
 func DefaultTxExecutor(_ context.Context,
 	blockSize int,
 	ms storetypes.MultiStore,
-	deliverTxWithMultiStore func(int, storetypes.MultiStore, map[string]any) *abci.ExecTxResult,
```

Comment:
can we keep supporting the usage in evm tx signature verification, just revert the cosmos-sdk native tx logic?

Reply:
I didn't notice sdk was outdated